src/main/java/nextstep/security/authorization/matcher/MvcRequestMatcher.java

```diff
@@ -0,0 +1,35 @@
+package nextstep.security.authorization.matcher;
+
+import jakarta.servlet.http.HttpServletRequest;
+import org.springframework.http.HttpMethod;
+
+import java.util.regex.Pattern;
+
+public class MvcRequestMatcher implements RequestMatcher {
+    private final HttpMethod method;
+    private final Pattern pattern;
+
+    public MvcRequestMatcher(HttpMethod method, String pattern) {
+        this.method = method;
+        this.pattern = compile(pattern);
+    }
+
+    private Pattern compile(String regex) {
+        return regex == null || regex.isBlank()
+                ? null
+                : Pattern.compile(regex);
```

Comment:
```suggestion
        if (regex == null || regex.isBlank()) {
            return null;
        }
        return Pattern.compile(regex);
```
개인적으로는 삼항연산자보다는 if문으로 early return 해주는 것이 한줄 한줄을 읽을 때 컨텍스트를 외워두지 않아도 되어서 읽기 편한 코드로 만들어준다고 생각합니다 :)

Reply:
[e23654e](https://github.com/next-step/spring-security-authorization/pull/9/commits/e23654eee8c18779f69bc3fd9099f82e1ac94bcd) 커밋에서 삼항연산자 대신 if 문을 사용하도록 모두 변경하였습니다!